src/uoapi/course/dependencies.py

```diff
@@ -0,0 +1,79 @@
+from typing import (
+    Generic,
+    TypeVar,
+    Optional,
+    List
+)
+
+
+T = TypeVar('T')
+
+
+class DependencyNode(Generic[T]):
+    kind = ''
+
+    def __init__(
+        self,
+        children: Optional[List["DependencyNode[T]"]] = None
+    ):
+        if children:
```

Comment:
```suggestion
        if children is not None:
```
I thought flake8 was supposed to catch [PEP-8 things](https://pep8.org/#programming-recommendations)?